syntax = "proto3";
package proto;

message BigVectorOfInts {
  repeated int32 int_list = 1;
}

message BigStrings {
    repeated bytes str_list = 1;
}

message VectorOfBigString {
    repeated string str_list = 1;
}

message Item {
    int32 uuid = 1;
    int32 cnt = 2;
    int32 ext = 3;
}

message GetBagInfoRsp {
    int32 retcode = 1;
    string msg = 2;
    repeated Item item_list = 3;
}